lib/presentation/screen/main_screen.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:star_book/presentation/shared/custom_bottom_nav_bar.dart';
+import 'package:star_book/widgets/gradient_scaffold.dart';
+
+class MainScreen extends StatefulWidget {
+  const MainScreen({super.key});
+
+  @override
+  State<MainScreen> createState() => _MainScreenState();
+}
+
+class _MainScreenState extends State<MainScreen> {
+  late int _selectedPage;
+  late final PageController pageController;
+
+  @override
+  void initState() {
+    super.initState();
+    _selectedPage = 0;
+    pageController = PageController(initialPage: _selectedPage);
+  }
+
+  void onPageChanged(int page) {
+    setState(() {
+      _selectedPage = page;
+    });
+  }
+
+  void onNavBarItemTapped(int page) {
+    pageController.jumpToPage(page);
+  }
+
+  List<Widget> pages = <Widget>[];
+
+  @override
+  Widget build(BuildContext context) {
+    return GradientScaffold(
+      body: Stack(
+        children: [
+          PageView(
+            controller: pageController,
+            onPageChanged: onPageChanged,
+            children: pages,
+          ),
+          CustomBottomNavBar(
+            backgroundColor: Colors.white,
+            selectedItemColor: const Color(0xFF7B7CFF),
```

Comment:
define color in theme

Reply:
I commented on this above.

Color will be implemented from the theme after completing this issue #298